auditbeat/module/file_integrity/schema/Source.go

```diff
@@ -26,16 +26,19 @@ type Source byte
 const (
 	SourceScan     Source = 0
 	SourceFSNotify Source = 1
+	SourceEBPF     Source = 2
 )
 
 var EnumNamesSource = map[Source]string{
 	SourceScan:     "Scan",
 	SourceFSNotify: "FSNotify",
+	SourceEBPF:     "EBPF",
```

Comment:
My opinion is that `eBPF` is more recognizable, even if the prior two values began with caps. Any reason not to switch it to `eBPF`?

Reply:
I thought it should be uppercase for consistency, no other reasons, I can change that

Reply:
How would this work for `Source`? `SourceeBPF` -> `"eBPF"` or `SourceEBPF` -> `"eBPF"`?
In case the former is better, should I also change `BackendEBPF` to `BackendeBPF`? (to me personally, this is less readable, but I have no strong opinion)